lib/krane/deploy_task.rb

```diff
@@ -238,9 +238,10 @@ def secrets_from_ejson
     def discover_resources
       @logger.info("Discovering resources:")
       resources = []
-      crds_by_kind = cluster_resource_discoverer.crds.group_by(&:kind)
+      crds_by_group_kind = cluster_resource_discoverer.crds.group_by(&:group_kind)
```

Comment:
Are there tests to cover this change?

Reply:
will be great to test this change, this is supposed to change the order of the items

Reply:
yeah, sorry, I misread the issue in https://github.com/Shopify/krane/issues/873 and I thought a consequence of this PR is that it'll change the order of the items in some list, however, this change is about grouping differently. Sorry for the noise